tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -26,10 +27,16 @@ ${MS_TABLE_STATUS_FAILURE}=    //span[contains(@class,"pf-c-icon__content")][con
 *** Keywords ***
 Open Model Serving Home Page
     [Documentation]    Verifies submenu Settings > "Model Serving" is visible and navigates to page
-    Page Should Contain    Model Serving
-    Click Link      Model Serving
-    Wait Until Page Contains    Manage and view the health and performance of your deployed models.    timeout=30
-    Wait Until Page Contains Element    ${MS_HEADER_XP}    timeout=30
+    SeleniumLibrary.Page Should Contain    Model Serving
+    SeleniumLibrary.Click Link      Model Serving
+    Sleep    1s
+    ${loading_error} =    Run Keyword And Return Status    SeleniumLibrary.Page Should Contain    Problem loading model serving page
```

Comment:
why that would happen?

Reply:
I've seen it happen while testing locally, sometimes the user would get an error page saying that they couldn't list the resources, and a refresh always fixed it

Reply:
hm interesting. Could you add a warn and a screenshot here? So next time we capture and report it (I believe it should not happen)